### ISSUE TITLE
Bug: take into account how maxmind DB changes over time

### ISSUE THREAD

**Issue by aaronkaplan** [cybergreen-net/etl-loader#7]
We need historic maxmind DBs for enrichment.

**Comment by kxyne** [comment]
Is the intention so we can determine which IP resolved to which country at the time of collection?

**Comment by chorsley** [comment]
Right, same for ASNs. The ETL pipeline currently only uses the latest dataset for CC and ASN, should be based on the dataset current at the date of the observation.

**Comment by aaronkaplan** [comment]
> On 22 Aug 2016, at 09:58, kxyne <notifications@github.com> wrote:
> 
> Is the intention so we can determine which IP resolved to which country at the time of collection?
> 

Yes.
